app/models/repository.rb

```diff
@@ -7,6 +7,7 @@ class Repository < ApplicationRecord
 
   scope :only_installer_updates, -> { where(installer_updates: true) }
   scope :only_mirrored, -> { where(mirroring_enabled: true) }
+  scope :only_fully_mirrored, -> { where(mirroring_enabled: true).where.not(last_mirrored_at: nil) }
```

Comment:
I took a look on where the `only_mirrored` scope is in use:

```ruby
lib/rmt/cli/repos.rb
101:    repositories = ((scope == :all) ? Repository.only_scc : Repository.only_scc.only_mirrored).order(:name, :description)

lib/rmt/cli/export.rb
14:    data = Repository.only_mirrored.inject([]) { |data, repo| data << { url: repo.external_url, auth_token: repo.auth_token.to_s } }

app/controllers/api/connect/v4/repositories/installer_controller.rb
9:        product.repositories.only_installer_updates.only_mirrored,

app/controllers/api/connect/v3/systems/products_controller.rb
94:    mirrored_repos = @product.repositories.only_enabled.only_mirrored
```

I'm not sure if I got it correctly, but it seems `only_mirrored` is not semantically correct, since it only returns repositories which are "mirroring enabled", not which were mirrored. The new scope you created looks more like `only_mirrored` than the existing one.

Do you think we would benefit from renaming it to, for example, `only_mirroring_enabled`? What are your thoughts?

Reply:
@lcaparroz I used your feedback in a new commit. I left the `only_fully_mirrored` scope to make the focus of this scope clear.

Reply:
Nice, LGTM!